graphql/introspection/introspection.go

```diff
@@ -299,3 +302,27 @@ func AddIntrospectionToSchema(schema *graphql.Schema) {
 
 	schema.Query = isQuery
 }
+
+// ComputeSchemaJSON returns the result of executing a GraphQL introspection
+// query.
+func ComputeSchemaJSON(schemaBuilderSchema schemabuilder.Schema) ([]byte, error) {
```

Comment:
It would be nice if this also worked on schema.Schema, but we can do that some other time.